PlantDiary21FS7024002/SpecimenSchema.json

```diff
@@ -24,8 +24,8 @@
       "properties": {
         "lat": {
           "type": "number",
-          "minimum": 50,
-          "maximum": 100
+          "minimum": -180,
+          "maximum": 180
```

Comment:
The range of latitude is -180 to + 180, not 50 to 100.
Source: https://www.fws.gov/stand/standards/de_latlon_WWW.html#:~:text=Values%20range%20from%20positive%20180,located%20at%20negative%2090%20degrees.

Reply:
But... are we really GPSing plants on Antarctica?  Maybe -90 to +90 is more appropriate.